okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -201,6 +201,13 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "    \"permissionsRequired\" : [ \"okapi.discovery.nodes.get\"  ], "
         + "    \"type\" : \"internal\" "
         + "   }, "
+        // import modules
+        + "   {"
+        + "    \"methods\" :  [ \"POST\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/import/modules\","
+        + "    \"permissionsRequired\" : [ \"okapi.import.modules.post\" ], "
```

Comment:
If someone has okapi.proxy.tenants.post permission and therefore is allowed to use POST /_/proxy/modules with one module at a time why should they not do exactly the same thing with a list of modules?
My suggestion is to avoid this new permission and reuse okapi.proxy.tenants.post.

Reply:
Not what we typically, do. But ok.